pkg/apis/operator/v1alpha1/openshiftpipelinesascode_defaults.go

```diff
@@ -20,7 +20,10 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/openshift-pipelines/pipelines-as-code/pkg/params/settings"
+	pacConfigutil "github.com/openshift-pipelines/pipelines-as-code/pkg/configutil"
```

Comment:
This is something on naming convention
We usually follow lowercase letter instead of camelCase

Reply:
I think this is fine for alias

Reply:
It is like that at multiple places